go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/onsi/gomega v1.18.0
 	github.com/openshift/api v3.9.1-0.20191111211345-a27ff30ebf09+incompatible
 	github.com/openshift/generic-admission-server v1.14.1-0.20210422140326-da96454c926d
+	github.com/openshift/library-go v0.0.0-20210916194400-ae21aab32431
```

Comment:
Assume we need to wait for idp-client-api change to get merged so you can pull it in with a change to go mod before merging this PR

Reply:
yeah thats right